.travis.yml

```diff
@@ -1,3 +1,4 @@
+dist: xenial   # required for Python >= 3.7
```

Comment:
pypy and pypy3 aren't available on xenial, so I think we need to specify an older version in the build matrix